raymon/tests/conftest.py

```diff
@@ -44,3 +46,16 @@ def envsecretfile(tmp_path):
     with open(tmp_file, "w") as f:
         f.write("client_secret")
     return tmp_file
+
+
+@pytest.fixture
+def load_data(dpath="raymon/tests/sample_data", lim=10):
```

Comment:
call this method `images` instead of load_data. As a fixture it is no longer a method but a variable. So its name should not indicate en action but an object.